### PR TITLE
Chore(fieldset): Reset fieldset properties

### LINKED ISSUE #280: Moai should reset fieldset properties
I think Moai should reset border, padding and margin of fieldset for preventing from breaking layout.

Screen Shot 2021-08-23 at 13 50 48

### PR DESCRIPTION
**Pull request by ro-vn**
**CHANGES**
- Fix #280

### CODE REVIEW

File: core/src/global/global.css [hunk 1]
```diff
@@ -129,6 +129,25 @@ th {
 	text-align: left;
 }
 
+legend {
+	display: table;
+	/* Correct `color` not being inherited in IE 8/9/10/11 */
+	color: inherit;
+	/* Remove padding so people aren't caught out if they zero out fieldsets */
+	padding: 0;
+}
```
Comment (thien-do) [inline core/src/global/global.css:138]: Hi, if you don't mind, may I ask where does this come from?

Comment (ro-vn) [reply core/src/global/global.css:138]: Hi, I got from this [normalize.css ](https://github.com/necolas/normalize.css/blob/master/normalize.css#L240)

Comment (thien-do) [reply core/src/global/global.css:138]: hm we did use modern-normalize.. not sure why they drop this..

**Comment by thien-do** [comment]
Sorry @ro-vn but I don't think we can move forward with this :( Moai should only have:

1) (modern)-normalization
2) no other global css

**Pull request closed without merging**